tests/OSPSuite.InstallationValidator.Tests/Presentation/MainPresenterSpecs.cs

```diff
@@ -35,49 +35,22 @@ protected override void Context()
 
          _applicationConfiguration = A.Fake<IApplicationConfiguration>();
          A.CallTo(() => _applicationConfiguration.IssueTrackerUrl).Returns(Constants.ISSUE_TRACKER_URL);
-         sut = new MainPresenter(_mainView, _dialogCreator, _batchStarterTask, _batchComparisonTask, _applicationConfiguration);
+         _validationReportingTask = A.Fake<IValidationReportingTask>();
+         sut = new MainPresenter(_mainView, _dialogCreator, _batchStarterTask, _batchComparisonTask, _applicationConfiguration, _validationReportingTask);
       }
    }
 
-   public class When_cancaling_a_validation_is_declined : concern_for_MainPresenter
```

Comment:
I don't have a way to force the dialog open for confirmation.

The boolean prevents it unless the validation task is running. Any idea about that?